### ISSUE TITLE
Implement Menu (new design pending)

### ISSUE THREAD

**Issue by thellmund** [serlo-org/serlo-abc#12]

**Comment by SimonKoehl** [comment]
small link: "Info für Lehrkräfte" oben links
small link: "Eine App von Serlo Education"

Neue Anordnung der Buchstaben bzw. Kapitel nach folgender Systematik:
(A)(N)(E)("Ich bin Anna")(Zeichen für "Wiederholung")
(S)(T)(O)("Hallo")(Zeichen für "Wiederholung")
...

Unten ist ein Footer mit AGBs, Partner. etc. (Ansprechpartner ist Ronnit)

**Comment by inyono** [comment]
> (A)(N)(E)("Ich bin Anna")(Zeichen für "Wiederholung")

will there be enough space for that?

**Comment by SimonKoehl** [comment]
yes, may be. I think it's no problem, when they take two rows as long as they are grouped in any way ..

**Comment by inyono** [comment]
depends on #77
